src/main/java/org/tailormap/api/controller/admin/TaskAdminController.java

```diff
@@ -78,13 +77,11 @@ public TaskAdminController(
       summary = "List all tasks, optionally filtered by type",
       description =
```

Comment:
there is a known issue with multiline string literals, see: https://github.com/diffplug/spotless/issues/1628 and https://github.com/palantir/palantir-java-format/issues/930

We don't uses these much, but it's not pretty

Reply:
Also read: https://jqno.nl/post/2024/08/24/why-are-there-no-decent-code-formatters-for-java/